### PR TITLE
docs: place download section in toh to the top

### PR DESCRIPTION
**Pull request by ajitsinghkaler**
this is part of a larger effort to standardise download sections on angular.io

This commit partially addresses #35459

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

No standardized download exampes

Issue Number: #35459

## What is the new behavior?
Standardized download examples

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by kapunahelewong — APPROVED** [review]
Thanks for this, @ajitsinghkaler! Looks good to me, but I'm also requesting review from another docs team member so we have consensus.

**Review by jbogarthyde — APPROVED** [review]
This looks like just the right thing to do.
I added the preview tag-- I'd like to see how it looks on the page.

File: aio/content/tutorial/toh-pt5.md [hunk 1]
```diff
@@ -13,6 +13,13 @@ When you’re done, users will be able to navigate the app like this:
   <img src='generated/images/guide/toh/nav-diagram.png' alt="View navigations">
 </div>
 
+<div class="alert is-helpful">
```
Comment (jbogarthyde) [inline aio/content/tutorial/toh-pt5.md:16]: After looking at the preview, I think you might consider putting this link above the image, to make it more visible at the top of the page.
I would put it right after the bullet list.

**Comment by ajitsinghkaler** [comment]
@jbogarthyde @kapunahelewong added the suggestions, please check

**Comment by jbogarthyde** [comment]
LGTM

**Comment by ajitsinghkaler** [comment]
@StephenFluin can you please review this

**Comment by ajitsinghkaler** [comment]
@StephenFluin Can you please review this

**Comment by ajitsinghkaler** [comment]
@jbogarthyde @kapunahelewong can we get this merged

**Comment by kapunahelewong** [comment]
Thanks for the tag, @ajitsinghkaler. I've requested @aikidave to review to help move this forward.

**Comment by ajitsinghkaler** [comment]
@aikidave @StephenFluin can you help me get this merged

**Review by aikithoughts — APPROVED** [review]

**Pull request closed without merging**